WordPress/src/main/java/org/xmlrpc/android/XMLRPCClient.java

```diff
@@ -534,6 +534,12 @@ private Object callXMLRPC(String method, Object[] params, File tempFile)
                 // Detect login issues and broadcast a message if the error is known
                 switch (e.getFaultCode()) {
                     case 403:
+                        // Ignore 403 error from certain methods known for replying with incorrect error code on
+                        // lacking permissions
+                        if ("wp.getPostFormats".equals(method) || "wp.getCommentStatusList".equals(method)
+                            || "wp.getPostStatusList".equals(method) || "wp.getPageStatusList".equals(method)) {
+                            break;
```

Comment:
This call is done in the [RefreshBlogContentTask](https://github.com/wordpress-mobile/WordPress-Android/blob/bac388aac9284360069856505ee1c89c713211d6/WordPress/src/main/java/org/xmlrpc/android/ApiHelper.java#L229-L229), which is call when we switch to a blog or when we the app starts. The call fails silently, and the user can only select the default post formats when he wants to create a new post.